Translations/en-GB.lproj/preferences.strings

```diff
@@ -183,7 +183,7 @@
 "PD2-iW-BGK.title" = "Enable home access so Keka can be used on all your user files, such as in your Desktop or Downloads folder.";
 
 /* Class = "NSButtonCell"; title = "Enable Notification Center integration"; ObjectID = "SOL-6U-5P9"; */
-"SOL-6U-5P9.title" = "Enable Notification Center integration";
+"SOL-6U-5P9.title" = "Enable Notification Centre integration";
```

Comment:
You've seen this one referenced anywhere in macOS? I don't think this should be translated:

https://support.apple.com/en-gb/HT204079

Reply:
My bad, just saw it in macOS